services/analysis-cached.js

```diff
@@ -19,7 +19,7 @@ const SQL_QUERIES = {
   lossFires:
     'SELECT {select_location}, umd_tree_cover_loss__year, SUM(umd_tree_cover_loss__ha) AS umd_tree_cover_loss__ha, SUM(umd_tree_cover_loss_from_fires__ha) AS "umd_tree_cover_loss_from_fires__ha" FROM data {WHERE} GROUP BY umd_tree_cover_loss__year, {location} ORDER BY umd_tree_cover_loss__year, {location}',
   lossFiresOTF:
-    'SELECT umd_tree_cover_loss__year, sum(umd_tree_cover_loss__ha), sum(umd_tree_cover_loss_from_fires__ha) FROM data WHERE umd_tree_cover_loss__year >= {startYear} AND umd_tree_cover_loss__year <= {endYear} AND umd_tree_cover_density_2000__threshold >= 30 GROUP BY umd_tree_cover_loss__year',
```

Comment:
huh, wonder why did we have a hardcoded value here 🤔

Reply:
Most likely an oversight that evaded Code Review and QA. I've added my thoughts as to how it may happened to the [JIRA](https://gfw.atlassian.net/browse/FLAG-900?focusedCommentId=20240) task. I would guess that during the development of a new widget, the example SQL query we get from the data team was used directly in order to build the widget, and then we missed the fact we need to do interpolation as well. 

We had a similar issue before, concerning the `{startYear}` and `{endYear}` addressed [here](https://github.com/wri/gfw/commit/98e45f571d5607b6b4eea3f327dc225878be7735).